File: cr-classifier/utils/response_utils.py
from flask import jsonify


def make_success_response(result):
    if result is None:
        return {
            'status': 'OK',
            'message': 'null',
            'result': ''
        }

    if result == True:
        return {
            'status': 'OK',
            'message': 'null',
            'result': 'true'
        }

    if result == False:
        return {
            'status': 'OK',
            'message': 'null',
            'result': 'false'
        }

    return jsonify({
        'status': 'OK',
        'message': 'null',
        'result': result
    })


def make_error_response(error_message):
    return {
        'status': 'Error',
        'message': error_message,
    }

File: cr-scout/src/utils/ResponseUtils.js
export const makeSuccessResponse = (res, result) => res.status(200).json({
  status: 'OK',
  message: null,
  result,
});

export const makeErrorResponse = (res, error) => res.status(500).json({
  status: 'Error',
  message: error.message,
  result: null,
});
